cdap-ui/app/cdap/components/PluginJSONCreator/Create/Content/OutputPage/ImplicitSchemaDefiner/index.tsx

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright © 2020 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import Button from '@material-ui/core/Button';
+import Alert from 'components/Alert';
+import WidgetWrapper from 'components/ConfigurationGroup/WidgetWrapper';
+import { useOutputState } from 'components/PluginJSONCreator/Create';
+import debounce from 'lodash/debounce';
+import * as React from 'react';
+
+export enum SchemaStatus {
+  Normal = 'Normal',
+  Success = 'SUCCESS',
+  Failure = 'FAILURE',
+}
+
+const ImplicitSchemaDefiner = () => {
+  const { schema, setSchema } = useOutputState();
+  const [localSchema, setLocalSchema] = React.useState(JSON.stringify(schema, undefined, 2));
+
+  const [schemaStatus, setSchemaStatus] = React.useState(SchemaStatus.Normal);
+
+  React.useEffect(() => {
+    if (schemaStatus === SchemaStatus.Success) {
+      // After a timeout for 500ms, reset schemaStatus to make a success Alert component disappear
+      debounce(() => setSchemaStatus(SchemaStatus.Normal), 1500)();
```

Comment:
This is not needed. https://github.com/cdapio/cdap/blob/develop/cdap-ui/app/cdap/components/Alert/index.js#L23 We already have the logic to close the success message after 3 seconds in the Alert component

Reply:
Made changes to use `onClose` instead of `debounce`